rgw/v2/tests/s3cmd/configs/test_large_object_gc.yaml

```diff
@@ -0,0 +1,8 @@
+# BZ id : 1892644#c52
+# script: test_large_object_gc.py
+config:
+ user_count: 1
+ bucket_count: 1
+ objects_count: 1
+ gc_verification: true
+ local_file_delete: false
```

Comment:
It is a good practice to have 2 spaces for nested items. Would it be possible to add an addition space for these lines.

Reply:
Done